x-pack/plugins/cloud_security_posture/server/routes/benchmark_rules/bulk_action/bulk_action.ts

```diff
@@ -10,6 +10,7 @@ import {
   CspBenchmarkRulesBulkActionRequestSchema,
   CspBenchmarkRulesStates,
   cspBenchmarkRulesBulkActionRequestSchema,
+  CspBenchmarkRulesBulkActionResponse,
 } from '../../../../common/types/rules/v3';
```

Comment:
any reason not to import from `latest`? now when we have v4 I'm lost in the version tbh :)